examples/statsmakie.jl

```diff
@@ -183,7 +183,7 @@
     end
 
     @cell "Errorbar" [errorbar] begin
-        using StatsMakie, Makie
```

Comment:
That's an important change... Is this ready to merge?

Reply:
Yep!

Reply:
You'll probably need to merge, I'm on my phone and Guthub doesn't let you merge a failing PR from a phone